src/core/Akka.Tests/Actor/Dispatch/Bug2640Spec.cs

```diff
@@ -76,6 +82,7 @@ public async Task ForkJoinExecutorShouldShutdownUponActorSystemTermination()
 
             threads = ReceiveN(100).Cast<Thread>().GroupBy(x => x.ManagedThreadId)
                 .ToDictionary(x => x.Key, grouping => grouping.First());
+            threads.Count.Should().Be(4, "Expected 4 distinct threads in this example");
```

Comment:
This change actually makes the test really flaky. In CI/CD environment, `Count` can either be 4 or 3, it looks like there is no guarantee that the system can provide all the requested thread count.